examples/menu_window_24bit.cpp

```diff
@@ -51,7 +51,6 @@ std::string render(Term::Window_24bit& scr,
 int main() {
     try {
         Terminal term(true, true, false);
-        Term::save_screen();
```

Comment:
Is the screen saved automatically?

Reply:
Yes, I have moved it into the constructor and destructor of the Terminal class. Basically the first `true` of it's parameters turns it on. I think it's much cleaner that way.

Reply:
Perfect.